src/commcare_cloud/ansible/roles/commcarehq/tasks/celery_cron.yml

```diff
@@ -0,0 +1,27 @@
+# To remove celery tasks that are running for more than 36 hours.
+#
+- name: Celery cron jobs for soft kill tasks running from last 36h
+  become: true
+  cron:
+    name: "Sending SIGTERM to celery processes running morethan 36h"
+    job: "kill -15 $(ps -eo comm,pid,etimes | awk '/celeryd:/ {if ($4 > 36*60*60) { print $3 }}')"
+    minute: 15
+    hour: "*"
+    weekday: "0,1,2,3,4,5,6"
+    user: root
+    cron_file: celery_sigterm
+  tags:
+    - cron
+
+- name: Cron job to Kill Celery tasks running from last 46h
+  become: true
+  cron:
+    name: "Sending SIGTERM to celery processes running morethan 46h"
+    job: "kill -9 $(ps -eo comm,pid,etimes | awk '/celeryd:/ {if ($4 > 46*60*60) { print $3 }}')"
```

Comment:
can we make these time limits configurable / support disabling these cron jobs?

Reply:
@prem-fissionhq I think that's a good suggestion from Simon. Could you move `46` to a variable called `celery_hours_before_hard_kill` and `36` to a variable called `celery_hours_before_soft_kill`? A value of `None` should cause the cron job to be removed (I think something like `state: "{{ 'present' if celery_hours_before_soft_kill else 'absent' }}` should do it.)

@snopoke I do think this is good default behavior, but I also could see ICDS not wanting to guinea pig it along with us, which would be totally reasonable. @prem-fissionhq let's set the default to None and then use these values (36, 46) for SaaS envs by setting them in the public.yml files. (I know that was a lot; feel free to offline with me)

Reply:
Sure @dannyroberts. I will make changes accordingly.

Reply:
Just following up to make sure this change is coming

Reply:
@snopoke yes, working on it.

Reply:
Updated the files which suggested changes. Creating new PR with the changes.